### PR TITLE
[addons] label2 cleanup

### PR DESCRIPTION
**Pull request by tamland**
Fixes the label2 values for addon listings. It should now use the label mask from sorting as we do everywhere else.

### CODE REVIEW

File: xbmc/addons/GUIWindowAddonBrowser.cpp [hunk 1]
```diff
@@ -369,14 +369,16 @@ bool CGUIWindowAddonBrowser::GetDirectory(const std::string& strDirectory, CFile
...
 
   for (int i = 0; i < items.Size(); ++i)
-    SetItemLabel2(items[i]);
+    UpdateStaus(items[i]);
 
   return result;
 }
 
-void CGUIWindowAddonBrowser::SetItemLabel2(CFileItemPtr item)
+void CGUIWindowAddonBrowser::UpdateStaus(const CFileItemPtr& item)
```
Comment (Razzeee) [inline xbmc/addons/GUIWindowAddonBrowser.cpp]: Typo? Or is Staus just something I don't know? :D

**Comment by tamland** [comment]
thank @Razzeee! jenkins build this please

**Comment by HitcherUK** [comment]
@tamland this breaks the 'Downloading nn%' label2.

**Comment by tamland** [comment]
@HitcherUK 'Breaks' how and where..?

**Comment by HitcherUK** [comment]
Add on browser when installing an add on. Label2 should show 'Downloading nn%' but it doesn't after this change.

**Pull request merged**